src/Voxel.MiddyNet.Tracing.Core/TraceContext.cs

```diff
@@ -92,5 +92,10 @@ private static string RandomString(int length)
             return new string(Enumerable.Repeat(chars, length)
                 .Select(s => s[random.Next(s.Length)]).ToArray());
         }
+
+        public static TraceContext ChangeParentId(TraceContext traceContext)
```

Comment:
Maybe we can call this Mutate, as it's called in the specification?

Reply:
Done